### ISSUE TITLE
[K8s-Stack] Add Global Variables for Image Registry and PullSecrets

### ISSUE THREAD

**Issue by alihoda** [VictoriaMetrics/helm-charts#818]
Is there any way to add global variables for image registry and pullSecrets in k8s stack? Just like what is in Prometheus charts.
```yaml
global:
  # global:
  #   imagePullSecrets:
  #   - name: pullSecret1
  #   - name: pullSecret2
  imagePullSecrets: []
  #
  # Allow parent charts to override registry hostname
  imageRegistry: ""
```
In the way that dependencies also use it.

**Comment by aureq** [comment]
@Haleygo The key point for this feature request is to be able to leverage private registries that can proxy to other registries. I use Harbor and it supports [caching and proxying](https://goharbor.io/docs/2.11.0/administration/configure-proxy-cache/) many other registries.

**Comment by matrix-root** [comment]
Doesn't work any more with latest version

**Comment by matrix-root** [comment]
Render - some nil map issue, vmks chart

**Comment by AndrewChubatiuk** [comment]
thanks for spotting this. fixed in 0.63.5
